Add docs for edge import mode. (#158)
memgraph/documentation: pages/fundamentals/storage-memory-usage.mdx

Value does not contain `indexing label`.

### Edge import mode

Memgraph on-disk storage supports `Edge import mode` to import edges with high throughput.

> In this mode user is allowed to create and modify edges only. Any vertex operations are prohibited and will through exception.

Reviewer: ```suggestion
In this mode, only edges can be created and modified. Any vertex operations are prohibited and will throw 
an exception.
```
